### PR TITLE
Work opi lab bringup - docs

### PR DESCRIPTION
**Pull request by mgheorghe**
- this is a draft intended for early feedback not intended to be merged as is.
- Please review and provide your feedback on content is 100% missing but you will like to see to better understand the lab.
- feedback on existing content is welcomed as well as contributions

### CODE REVIEW

File: lab/README.md
Comment (pdp2shirts) [inline lab/README.md:1]: I will add links to F5 BIG-IP-01 and -02 VPN README.MD when I have time to do that

File: lab/hardware/A100G/arista.config
Comment (pdp2shirts) [inline lab/hardware/A100G/arista.config:1]: looks good to me

File: lab/hardware/A100G/README.md
Comment (pdp2shirts) [inline lab/hardware/A100G/README.md:1]: lgtm

File: lab/hardware/KVM/README.md
Comment (pdp2shirts) [inline lab/hardware/KVM/README.md:1]: lgtm link works

File: lab/images/opi-lab-cabling.drawio.svg
Comment (pdp2shirts) [inline lab/images/opi-lab-cabling.drawio.svg:1]: do the different colored lines have specific meaning?

File: lab/hardware/TS/README.md
Comment (pdp2shirts) [inline lab/hardware/TS/README.md:1]: link works LGTM

File: lab/hardware/dh1/README.md
Comment (pdp2shirts) [inline lab/hardware/dh1/README.md:1]: until we get a dedicated management server. I have installed in the /certautomation directory on this server a container that uses SWAG to check the wildcard ssl certificate I have purchased for the lab. "*.opiproject-lab.org" There is also a container that runs ansible process to check for renewed certificates and uploads them automatically to the BIG-IPs. Do you want me to document that in this file?

File: lab/README.md
Comment (pdp2shirts) [inline lab/README.md:1]: I have purchased a domain from cloud flare we can use for the lab. The domain is "opiproject-lab.org" I have a few DNS entries created for "vpn.opiproject-lab.org" "big-ip-01.opiproject-lab.org", and "big-ip-02.opiproject-lab.org" I am thinking we add a section to this file where we link to a new file that describes the dns domain. thoughts?

File: lab/hardware/ips.md
Comment (pdp2shirts) [inline lab/hardware/ips.md:1]: once merged I can add the BIG-IP interface IPs to this file.

File: lab/hardware/nPDU/README.md
Comment (pdp2shirts) [inline lab/hardware/nPDU/README.md:1]: link works LGTM

**Review by pdp2shirts — APPROVED** [review]
I left a number of comments. This looks good to me and I would be fine if we merged it. I will make additions to it once merged.

**Review by glimchb — APPROVED** [review]
a lot of changes, better to split into multiple PRs next time

**Pull request merged**